background.js

```diff
@@ -1,14 +1,24 @@
-chrome.runtime.onInstalled.addListener(function() {
-    chrome.declarativeContent.onPageChanged.removeRules(undefined, function() {
+/*
+ * Add page match rules on install
+ * 
+ * When the url contains (fireworkscrm) or (pull-requests and the page contains
+ * the PR description field), then we enable the extension icon
+ */
+chrome.runtime.onInstalled.addListener(function () {
+    chrome.declarativeContent.onPageChanged.removeRules(undefined, function () {
         chrome.declarativeContent.onPageChanged.addRules([
-                {
-                    conditions: [
-                        new chrome.declarativeContent.PageStateMatcher({
-                            pageUrl: { urlContains: 'fireworkscrm.' },
-                        })
-                    ],
-                    actions: [ new chrome.declarativeContent.ShowPageAction() ]
-                }
+            {
+                conditions: [
+                    new chrome.declarativeContent.PageStateMatcher({
+                        pageUrl: { urlContains: 'fireworkscrm.' },
+                    }),
+                    new chrome.declarativeContent.PageStateMatcher({
+                        pageUrl: { pathContains: 'pull-requests' },
```

Comment:
can we do both `urlContains` and `pathContains` in the same rule here? Coul dcall out the `gotoextinguisher` url part!

Reply:
updated to look for stash.gotoextinguisher.com